src/dynout_parser.cc

```diff
@@ -0,0 +1,135 @@
+/* Generated by re2c 1.3 */
+// Copyright 2011 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dynout_parser.h"
+
+#include "debug_flags.h"
+#include "metrics.h"
+
+#include <errno.h>
+
+using namespace std;
```

Comment:
Please do not use `using namespace std` anymore, we now want all new code to use `std::` prefixes.

Reply:
Removed `using namespace std`.